src/app.ts

```diff
@@ -593,6 +595,37 @@ const disableMfa = async (name?: string): Promise<void> => {
   console.log(chalk.green(`Disabled MFA on profile '${profileName}'`));
 };
 
+const timedOutStatusCheck = async (): Promise<void> => {
+  const delay = promisify(setTimeout);
+
+  await delay(2000);
+};
+
+const status = async (): Promise<void> => {
+  console.log(chalk.yellow(`Checking status...`));
+
+  const sts = new STS();
+
+  const response = await Promise.race([sts.getCallerIdentity().promise(), timedOutStatusCheck()]);
+
+  if (response) {
+    try {
```

Comment:
I don't think this try/catch is doing anything. I don't think anything in the try will ever throw.